aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -282,17 +282,18 @@ Tensor empty_like(
           .merge_in(options_)
           .merge_in(TensorOptions().memory_format(optional_memory_format));
```

Comment:
When `options.layout()` is `kSparse`, I think we should avoid adding `memory_format` to `options` because it is invalid for non-strided tensors.

Reply:
Thanks for the comment, OK I see. Let's bypass using `optional_memory_format`  as in C++ `zeros_like`.